engine/src/main/java/org/destinationsol/game/SolCam.java

```diff
@@ -36,6 +35,11 @@
     private static final float MOVE_SPD = 3f;
     private static final float MAX_SHAKE = .07f;
     private static final float SHAKE_DAMP = MAX_SHAKE;
+    public static boolean DIRECT_CAM_CONTROL = false;
+    public static SolUiControl controlUp = null;
```

Comment:
When using controls from normal bindings, direct camera control is uncontrollable when using combined/mouse controls (like that the screen starts moving left in constant speed). This should work whether you are using keyboard as well as combined controls. It's possible that this could be rather included in control schemes and drawn from them, but since I have written this code so far in the past (a week ago at least), it is equally possible that there is something forbiding from doing that as well as that I have just not thought of this that time. I'll need to give that more proper examination once I got the time.